### DISCUSSION TITLE
Logs on another accont without credentials

### DISCUSSION THREAD

**Discussion by megatonix** [nextcloud/all-in-one#3233] [Questions] [answered]
### Steps to reproduce
1.I was log into my account 
2. I refresh my browser
3. Now i'm log in to coworker account

### Expected behavior 

This shouldn't be happend, how i was able to log as another account by only refresh page?
My second coworker do the same thing, refresh page and log into the same account like me

### Actual behavior 

I can see Talk conversation, files and other stuff 

### Host OS 
Linus Ubuntu 22.04
 
#### Nextcloud AIO version 
v7.0.0
#### Current channel 
latest

**Comment by szaimen** [comment]
Hi, do you share the same PC and browser between users?

**Reply by megatonix** [reply]
No, each of us have different laptop and different browser

**Reply by szaimen** [reply]
This is interesting! I have not seen the problem yet. Can you deinstall all non-default Nextcloud apps and check if it fixes the problem?

**Reply by megatonix** [reply]
Yes, this is very very strange and interesting. I will try do that but it hard to reproduce becouse this "bug" work like 5 min...

**Comment by szaimen** [comment, marked as answer]
Hi, are you running AIO behind NPM? If so, did you enable `Cache Assets` in your config? This would explain this behaviour. See https://github.com/nextcloud/server/issues/40863
